Vagrantfile

```diff
@@ -8,12 +8,14 @@ gopath_folder="/opt/gopath"
 
 cluster_ip_nodes = ""
 
-provision_common_once = <<SCRIPT
+provision_common = <<SCRIPT
```

Comment:
No, those are not my changes.. they got added extra..
On Jul 10, 2016 4:17 PM, "jojimt" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/contiv/netplugin/pull/444#discussion_r70190230>:
>
> > @@ -8,12 +8,14 @@ gopath_folder="/opt/gopath"
> >
> >  cluster_ip_nodes = ""
> >
> > -provision_common_once = <<SCRIPT
> > +provision_common = <<SCRIPT
>
> Are these changes to Vagrantfile intentional? They don't seem to be.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/contiv/netplugin/pull/444/files/7d1cbb78f7ca15e74c227963b54e04a716a1cec3#r70190230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASwtyfQ9tEd1dN55yTCePnFmIL6cythOks5qUX1ygaJpZM4JGrpk>
> .
>